tests/components/zwave/test_binary_sensor.py

```diff
@@ -4,7 +4,7 @@
 from unittest.mock import patch
 
 from homeassistant.components.zwave import const
-from homeassistant.components.binary_sensor import zwave
+import homeassistant.components.zwave.binary_sensor as zwave
```

Comment:
It's confusing when reading the test to call the platform just `zwave`. It would be more clear to call it `zwave_binary_sensor`.